sdk/create-dapp/templates/react-client-dapp/index.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<html lang="en" class="dark-theme" style="color-scheme: dark">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Sui dApp Starter</title>
+
+    <style>
```

Comment:
Thoughts on just incorporating Tailwind here? It's opinionated but I think most people will add it quickly.

Reply:
do we need these separate resets? Radix Themes provides minimal resets for all of the provided components it offers